RecoMET/METProducers/src/HcalNoiseInfoProducer.cc

```diff
@@ -127,15 +128,72 @@ HcalNoiseInfoProducer::HcalNoiseInfoProducer(const edm::ParameterSet& iConfig) :
   // adc -> fC for qie8 with PMT input, for laser monitor
   // Taken from Table 2 in 
   // https://cms-docdb.cern.ch/cgi-bin/DocDB/RetrieveFile?docid=3275&filename=qie_spec.pdf&version=1
-  adc2fCHF = std::vector<float> {-3,-0.4,2.2,4.8,7.4,10,12.6,15.2,17.8,20.4,23,25.6,28.2,30.8,33.4,
-                                 36,41.2,46.4,51.6,56.8,62,67.2,73,80.8,88.6,96.4,104,114.4,124.8,135,
-                                 148,161,150,163,176,189,202,215,228,241,254,267,280,293,306,319,332,
-                                 343,369,395,421,447,473,499,525,564,603,642,681,733,785,837,902,967,
-                                 902,967,1032,1097,1162,1227,1292,1357,1422,1487,1552,1617,1682,1747,
-                                 1812,1877,2007,2137,2267,2397,2527,2657,2787,2982,3177,3372,3567,
-                                 3827,4087,4347,4672,4997,4672,4997,5322,5647,5972,6297,6622,6947,
-                                 7272,7597,7922,8247,8572,8897,9222,9547,10197,10847,11497,12147,
-                                 12797,13447,14097,15072,16047,17022,17997,19297,20597,21897,23522,25147};
+  //adc2fCHF = std::vector<float> {-3,-0.4,2.2,4.8,7.4,10,12.6,15.2,17.8,20.4,23,25.6,28.2,30.8,33.4,
+  //                               36,41.2,46.4,51.6,56.8,62,67.2,73,80.8,88.6,96.4,104,114.4,124.8,135,
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
Indeed, I neglected to remove it.  I've pushed the change